### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by Jawad-Nazari**
### In this PR, I did the following:

- Created new tables for owners and species
- Modified animals table
- Inserted data into owners and species tables
- Inserted and modified owner_id and species_id into animal's table
- Wrote queries using JOIN to select data from multiple tables.

**What animals belong to Melody Pond?**
q1

**List of all animals that are pokemon (their type is Pokemon).**
q2

**List all owners and their animals, remember to include those that don't own any animal.**
q3

**How many animals are there per species?**
q4

**List all Digimon owned by Jennifer Orwell.**
q5

**List all animals owned by Dean Winchester that haven't tried to escape.**
q6

**Who owns the most animals?**
q7

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -14,3 +14,65 @@ INSERT INTO animals VALUES (5,'Charmander', '2020-02-08', 0, false, -11),
...
+All other animals are Pokemon
+Modify your inserted animals to include owner information (owner_id):
+Sam Smith owns Agumon.
+Jennifer Orwell owns Gabumon and Pikachu.
+Bob owns Devimon and Plantmon.
+Melody Pond owns Charmander, Squirtle, and Blossom.
+Dean Winchester owns Angemon and Boarmon.
+Remember these insertions and modifications happen in data.sql file. */
+
+INSERT INTO species (name)
```
Comment (Davidosky007) [inline data.sql:48]: * Kindly Write queries (using JOIN) to answer the following questions as there are [required ](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=Write%20queries%20(using%20JOIN)%20to%20answer%20the%20following%20questions%3A)  

> What animals belong to Melody Pond?
> List of all animals that are pokemon (their type is Pokemon).
> List all owners and their animals, remember to include those that don't own any animal.
> How many animals are there per species?
> List all Digimon owned by Jennifer Orwell.
> List all animals owned by Dean Winchester that haven't tried to escape.
> Who owns the most animals?
> Remember all these should be written in queries.sql file.

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Jawad-Nazari 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
*  Primary keys & foreign keys mechanism used for joining tables 👍 
* Multiple tables queried 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (Jawad-Nazari) [reply data.sql:48]: Hi @Davidosky007 ,

Thanks for your time and reviewing my code. I have pushed the queries for using JOIN. 🙏

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @Jawad-Nazari, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**